.github/workflows/tower_lsf_full_tests.yml

```diff
@@ -0,0 +1,31 @@
+name: nf-core LSF full size tests
+
+on:
+  pull_request:
+    branches:
+      - main
+      - dev
```

Comment:
> I only wanted to run this when merged into dev or main. Because when we open a PR, we have a conversation and there are multiple commits added along the way, so running full tests for each of these feels like a waste of resources and would overlap too. Does that sound okay?

which means you only can find the problems after being merged. is it too late?

the main branch is rather stable branch, we don't make PR against it much. We can only run this on main branch?

Reply:
The unit tests will be run so the problems should be highlighted either way. But if resources are not a problem, I don't mind running it before the merge. I think `nf-core` only does it when they release and it works for them.

Reply:
updated.